engine/src/test/java/com/arcadedb/query/sql/executor/AlterTypeExecutionTest.java

```diff
@@ -102,4 +102,11 @@ public void sqlAlterTypeCustom() {
     Assertions.assertNull(database.getSchema().getType("Suv").getCustomValue("age"));
     Assertions.assertFalse(database.getSchema().getType("Suv").getCustomKeys().contains("age"));
   }
+
+  @Test
+  public void sqlAlterTypeName() {
+    database.command("sql", "CREATE VERTEX TYPE Mpv");
```

Comment:
Could you please create a property with an index, so see if everything works as expected? Thanks in advance @daffodilistic !